frontend/src/pages/Screening.vue

```diff
@@ -151,17 +148,6 @@ export default {
   computed: {
     ...mapGetters(["screeningStudents"]),
     ...mapState(['screeningSuccess']),
-    tableData() {
-      return this.screeningStudents.map((student) => {
-        return {
-          StudentID: `${student.StudentID}`,
-          NativeLanguage: `${student.nativeLanguage.NativeLanguage}`,
-          EnglishProficiency: `${student.EnglishProficiency}`,
-          PhoneNumber: `${student.PhoneNumber}`,
-          FullName: `${student.FullName}`,
-        };
-      });
-    },
```

Comment:
Don't we still need this? Because we need to pull out `student.nativeLanguage.NativeLanguage`?

Reply:
sorry about that! Just fixed it by reinstating tableData, but where studentID remains an integer instead of a string. I changed the source of data for the table to screeningStudents and didn't realise it'd remove NativeLanguage.
The issue was with StudentID being turned into a string, which meant that it couldn't be patched to the backend as backend expects an integer.